synapse/storage/devices.py

```diff
@@ -825,6 +841,9 @@ def _add_device_change_txn(self, txn, user_id, device_ids, hosts, stream_id):
                     "device_id": device_id,
                     "sent": False,
                     "ts": now,
+                    "context": json.dumps(context)
+                    if opentracing.whitelisted_homeserver(destination)
+                    else "{}",
```

Comment:
given that this column is nullable, can we just:

```suggestion
                    else None,
```

Reply:
Sure